lib/configuration/configuration.ts

```diff
@@ -15,6 +15,14 @@ interface CompilerOptions {
   deleteOutDir?: boolean;
 }
 
+interface GenerateOptions {
+  spec?:
+    | boolean
+    | {
+        [key: string]: boolean;
+      };
```

Comment:
What about using `Record<string, boolean>` instead?

Reply:
I guess it could be used instead, however I don’t really see the real benefit of changing it to that instead? Could you explain?

Reply:
This type is predefined and comes with TypeScript (so there is no reason to define a new one) + we're using `Record` a lot in all Nest packages, hence, it would be nice to stay cohesive

Reply:
Ah! Yes, will commit a change for this later.

Reply:
Thank you :)